frontend/src/metabase/app-embed-sdk.tsx

```diff
@@ -5,13 +5,14 @@ import {
   EMBEDDING_SDK_IFRAME_EMBEDDING_CONFIG,
 } from "metabase/embedding-sdk/config";
 
-// Enable SDK mode for new iframe embedding.
-// Note that this is also defined in the SDK's entry point.
+/**
+ * Configuration overrides for simple embedding.
+ */
+EMBEDDING_SDK_IFRAME_EMBEDDING_CONFIG.isSimpleEmbedding = true;
 EMBEDDING_SDK_CONFIG.isEmbeddingSdk = true;
 EMBEDDING_SDK_CONFIG.metabaseClientRequestHeader = "embedding-simple";
-
-// Use the iframe embedding auth flow instead of the regular auth flow.
-EMBEDDING_SDK_IFRAME_EMBEDDING_CONFIG.isSdkIframeEmbedAuth = true;
+EMBEDDING_SDK_CONFIG.enableEmbeddingSettingKey = "enable-embedding-simple";
+EMBEDDING_SDK_CONFIG.tokenFeatureKey = "embedding_simple";
```

Comment:
I moved the configuration for simple embedding here so they're all in one place.